src/components/message/Message.js

```diff
@@ -67,12 +67,16 @@ Message.propTypes = {
   /** Dismiss button type */
   dismissType: PropTypes.oneOf(['string', 'icon']),
 
+  /** Dismiss text for dismissType of string */
```

Comment:
Small nitpick: This is now also the text-alternative for the button, even when it's just an icon.

Reply:
Do you think I should update this description or separate the label into its own prop?